core/primitives/benches/state_witness_borsh.rs

```diff
@@ -0,0 +1,67 @@
+//! Benchmark for Borsh encoding/decoding of ChunkStateWitness.
+//!
+//! Run with `cargo bench --bench state_witness_borsh`
+
```

Comment:
does changing the allocator make any difference? it won't be included by default & I doubt it but maybe worth checking.